linters/osv-scanner/plugin.yaml

```diff
@@ -0,0 +1,34 @@
+version: 0.1
+lint:
+  downloads:
+    - name: osv-scanner
+      version: 1.2.0
+      executable: true
+      downloads:
+        - os:
+            linux: linux
+            macos: darwin
+          cpu:
+            x86_64: amd64
+            arm_64: arm64
+          url: https://github.com/google/osv-scanner/releases/download/v${version}/osv-scanner_${version}_${os}_${cpu}
+  definitions:
+    - name: osv-scanner
+      files: [osvlockfile]
```

Comment:
nit: change name of file type to osv-lockfiles. I read current as a lockfile used for osv